RtAudio.h

```diff
@@ -1064,6 +1064,8 @@ class RtApiWasapi : public RtApi
   static DWORD WINAPI stopWasapiThread( void* wasapiPtr );
   static DWORD WINAPI abortWasapiThread( void* wasapiPtr );
   void wasapiThread();
+
+  static bool iaudioclient3_available;
```

Comment:
adding static variable here will not affect abi since it's never allocated on the stack.